src/components/layers.js

```diff
@@ -19,35 +21,22 @@ export const Layers = {
  */
 AFRAME.registerComponent("layers", {
   schema: {
```

Comment:
We technically had this exposed in gltf-component-mappings but it is not used by Spoke or the Blender Exporter so it is *highly* unlikely anyone would have used it for anything. Simplifying the schema to more directly map to the counterpart in BitECS land.